ryu/d2s_intrinsics.h

```diff
@@ -193,6 +193,7 @@ static inline uint32_t pow5Factor(uint64_t value) {
   const uint64_t n_div_5 = 3689348814741910323u;  // #{ n | n = 0 (mod 2^64) } = 2^54 / 5
   uint32_t count = 0;
   for (;;) {
+    assert(value != 0);
```

Comment:
I've put the `assert` back where it was. However, with this new implementation it doesn't make too much sense to have it inside the loop since `value` is updated by multiplication with a non zero number. It does make sense to have it outside the loop though. I leave it for the owner to decide.